src/routes/hooks/hooks-cache.spec.ts

```diff
@@ -308,11 +161,8 @@ describe.skip('Post Hook Events for Cache (Unit)', () => {
       }
     });
 
-    await request(app.getHttpServer())
-      .post(`/hooks/events`)
-      .set('Authorization', `Basic ${authToken}`)
-      .send(data)
-      .expect(202);
+    const cb = queuesApiService.subscribe.mock.calls[0][1];
```

Comment:
Question: Is there a clearer way to access the queue callback? I’m finding it difficult to understand what indices 0 and 1 represent.